src/components/FeatureBlock/FeatureBlock.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Text } from "../Text/Text";
+import "./FeatureBlockStyles.scss";
+import { FeatureBlockProps } from "./FeatureBlockTypes";
+
+export const FeatureBlock: React.FC<FeatureBlockProps> = ({
+  iconSrc,
+  title,
+  description,
+}) => (
+  <div className="feature-block">
+    <img src={iconSrc} className="feature-block__img" />
```

Comment:
Is the BEM style approach more of a personal preference or something that's necessary for namespacing CSS? I just took a look and it seems like the BEM style approach is necessary since I'm not seeing our CSS classes get hashed (lmao rip). I need to go back and edit some of the files I contributed.

Reply:
Just more of a personal preference! Hadn't even noticed the classes weren't getting hashed